config/secrets.yml.template

```diff
@@ -1,18 +1,20 @@
 # Sample secrets.yml file.
```

Comment:
I'm wondering if it's worth including a note that, if an environment doesn't have a secret value, to just delete that line. But that's definitely a nit not a blocker!